doc/source/dev/gitwash/gitwash.rst

```diff
@@ -0,0 +1,34 @@
+.. _using-git:
+.. _git-development:
+
+=====================
+ Git for development
+=====================
```

Comment:
NumPy removed the Development Workflow documentation from this list (https://www.numpy.org/devdocs/dev/gitwash/index.html) so I followed suit. It feels sort of incomplete without it, though.